modules/post/android/local/koffee.rb

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+require 'readline'
```

Comment:
This shouldn't be necessary any more since you're using the actions now instead of a REPL.

Reply:
'require' removed and sent video to the email msfdev@metaspolit.com with object "KOFFEE commit, CVE-2020-8539 - video updated with actions".